dune-project

```diff
@@ -1,13 +1,25 @@
-(lang dune 3.5)
+(lang dune 3.19)
```

Comment:
Check this one with Mark, I remember there were some reasons not to upgrade but those may be obsolete now.